src/components/Toggle/README.md

```diff
@@ -0,0 +1,261 @@
+---
+name: Toggle
+category: Forms
+platforms:
+  - web
+keywords:
+  - toggle
+  - switch
+  - selection
+  - choices
+  - options
+  - pick
+  - single selection form
+  - choice form
+  - option button
+  - enable
+  - disable
+---
+
+# Toggle
+
+Use to present a single choice to enable or disable a setting. Toggle is best used when multiple settings can be enabled or disabled and then saved all at once.
+
+---
+
+## Best practices
+
+Toggle should:
+
+- Be used when there are only two options available (enabled or disabled) and not a range of selections (zero, one, or multiple).
+- Be used to switch a single setting on and off.
+- Always be used with an associated label component.
+
+---
+
+## Content guidelines
+
+### Toggle labels
+
+Toggle labels should:
+
+- Start with a capital letter
+
+<!-- usagelist -->
+
+#### Do
+
+- Off
+
+#### Don’t
+
+- off
+
+<!-- end -->
+
+- Not end in punctuation if it’s a single sentence, word, or a fragment
+
+<!-- usagelist -->
+
+#### Do
+
+- Off
+
+#### Don’t
+
+- Off;
+
+<!-- end -->
+
+---
+
+## Examples
+
+### Default toggle
+
+Use toggle to let merchants enable or disable a setting.
+
+```jsx
+class ToggleExample extends React.Component {
+  state = {
+    visaChecked: true,
+    mcChecked: false,
```

Comment:
I'd like to see these, and the handlers use a more consistent naming convention. Something like `masterCardChecked` and `toggleMasterCardToggle`, since you use `visa` in the other ones.